library/sonarqube

```diff
@@ -1,22 +1,21 @@
-Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
-             Julien Lancelot <julien.lancelot@sonarsource.com> (@julienlancelot),
-             Christophe Levis <christophe.levis@sonarsource.com> (@christophelevis),
+Maintainers: Christophe Levis <christophe.levis@sonarsource.com> (@christophelevis),
              Pierre Guillot <pierre.guillot@sonarsource.com> (@pierre-guillot-sonarsource),
-             Michal Duda <michal.duda@sonarsource.com> (@michal-duda-sonarsource)
+             Tobias Trabelsi <tobias.trabelsi@sonarsource.com> (@tobias-trabelsi-sonarsource),
+             Lukasz Jarocki <lukasz.jarocki@sonarsource.com> (@lukasz-jarocki-sonarsource)
```

Comment:
Welcome new maintainers! @tobias-trabelsi-sonarsource and @lukasz-jarocki-sonarsource. Thank you to the previous maintainers, @janos-ss, @julienlancelot, @michal-duda-sonarsource!